From 9b7ef113f2ad2a96195d1392feffd013e7bce975 Mon Sep 17 00:00:00 2001
From: Adithya R <gh0strider.2k18.reborn@gmail.com>
Date: Thu, 28 Dec 2023 06:22:25 +0530
Subject: [PATCH 24/28] Launcher3: Use folder background color for QSB

Background color is black when black theme is used. That looked bad
on QSB for obvious reasons.

Change-Id: I4733c4a3429b38c46328d4c8682760e98655aa8f
---
 res/values/styles.xml | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/res/values/styles.xml b/res/values/styles.xml
index 8c5c350561..3f9e0eaa57 100644
--- a/res/values/styles.xml
+++ b/res/values/styles.xml
@@ -166,7 +166,7 @@
     <!-- A derivative project can extend these themes to customize the application theme without
          affecting the base theme -->
     <style name="AppTheme" parent="@style/LauncherTheme">
-        <item name="qsbFillColor">@*android:color/background_device_default_light</item>
+        <item name="qsbFillColor">@color/folder_background_light</item>
         <item name="qsbIconTintPrimary">@color/qsb_primary</item>
         <item name="qsbIconTintQuaternary">@color/qsb_quaternary</item>
         <item name="qsbIconTintSecondary">@color/qsb_secondary</item>
@@ -174,7 +174,7 @@
     </style>
 
     <style name="AppTheme.DarkMainColor" parent="@style/LauncherTheme.DarkMainColor">
-        <item name="qsbFillColor">@*android:color/background_device_default_light</item>
+        <item name="qsbFillColor">@color/folder_background_light</item>
         <item name="qsbIconTintPrimary">@color/qsb_primary</item>
         <item name="qsbIconTintQuaternary">@color/qsb_quaternary</item>
         <item name="qsbIconTintSecondary">@color/qsb_secondary</item>
@@ -182,7 +182,7 @@
     </style>
 
     <style name="AppTheme.DarkText" parent="@style/LauncherTheme.DarkText">
-        <item name="qsbFillColor">@*android:color/background_device_default_light</item>
+        <item name="qsbFillColor">@color/folder_background_light</item>
         <item name="qsbIconTintPrimary">@color/qsb_primary</item>
         <item name="qsbIconTintQuaternary">@color/qsb_quaternary</item>
         <item name="qsbIconTintSecondary">@color/qsb_secondary</item>
@@ -190,7 +190,7 @@
     </style>
 
     <style name="AppTheme.Dark" parent="@style/LauncherTheme.Dark">
-        <item name="qsbFillColor">@color/surface</item>
+        <item name="qsbFillColor">@color/folder_background_dark</item>
         <item name="qsbIconTintPrimary">@color/qsb_primary</item>
         <item name="qsbIconTintQuaternary">@color/qsb_quaternary</item>
         <item name="qsbIconTintSecondary">@color/qsb_secondary</item>
@@ -198,14 +198,14 @@
     </style>
 
     <style name="AppTheme.Dark.DarkMainColor" parent="@style/LauncherTheme.Dark.DarkMainColor">
-        <item name="qsbFillColor">@color/surface</item>
+        <item name="qsbFillColor">@color/folder_background_dark</item>
         <item name="qsbIconTintPrimary">@color/qsb_primary</item>
         <item name="qsbIconTintQuaternary">@color/qsb_quaternary</item>
         <item name="qsbIconTintSecondary">@color/qsb_secondary</item>
         <item name="qsbIconTintTertiary">@color/qsb_tertiary</item>
     </style>
     <style name="AppTheme.Dark.DarkText" parent="@style/LauncherTheme.Dark.DarkText">
-        <item name="qsbFillColor">@color/surface</item>
+        <item name="qsbFillColor">@color/folder_background_dark</item>
         <item name="qsbIconTintPrimary">@color/qsb_primary</item>
         <item name="qsbIconTintQuaternary">@color/qsb_quaternary</item>
         <item name="qsbIconTintSecondary">@color/qsb_secondary</item>
-- 
2.51.1

